base/linalg/special.jl

```diff
@@ -4,11 +4,12 @@
 
 # Interconversion between special matrix types
 convert(::Type{Bidiagonal}, A::Diagonal{T}) where {T} =
```

Comment:
Is the `where {T}` now superfluous? And likewise below?

Reply:
No, nice catch.